<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseForm</class>
 <widget class="QDialog" name="ExpenseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>577</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Expenses</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../app/icons.qrc">
    <normaloff>:/expense</normaloff>:/expense</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;MS Shell Dlg&quot;; </string>
  </property>
  <widget class="QTableView" name="view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>681</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/delete</normaloff>:/delete</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>691</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>ADD EXPENSE:</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>20</y>
      <width>634</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="date_line_edit">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>ddmmyyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amount">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Info:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="info_lineedit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="add_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../app/icons.qrc">
         <normaloff>:/new</normaloff>:/new</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>40</y>
     <width>357</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Partner:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="partner_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>80</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="document_date_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>22</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Document:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="document_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>147</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Spent:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="spent_line_edit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>date_line_edit</tabstop>
  <tabstop>amount</tabstop>
  <tabstop>info_lineedit</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>spent_line_edit</tabstop>
  <tabstop>partner_line_edit</tabstop>
  <tabstop>document_line_edit</tabstop>
  <tabstop>document_date_line_edit</tabstop>
  <tabstop>view</tabstop>
  <tabstop>delete_button</tabstop>
 </tabstops>
 <resources>
  <include location="../app/icons.qrc"/>
 </resources>
 <connections/>
</ui>
